xml/favicons.xml

```diff
@@ -473,10 +482,10 @@
   </fingerprint>
 
   <fingerprint pattern="^3ef81fad2a3deaeb19f02c9cf67ed8eb$">
-    <description>DocuWiki</description>
+    <description>DocuWiki (0.0.20080505)</description>
```

Comment:
Similar to the above; it feels like we can't mark this as version `0.0.20080505` if the application has used the same favicon for 13 years

```
curl -s -L https://raw.githubusercontent.com/dokuwiki/dokuwiki/master/lib/tpl/dokuwiki/images/favicon.ico | md5sum
3ef81fad2a3deaeb19f02c9cf67ed8eb  -
```

Let me know if I've misunderstood anything :+1:

Reply:
Apologies, I will double check these updated fingerprints and fix them accordingly. Thanks for pointing these out!